api/nakadi-event-bus-api.yaml

```diff
@@ -150,11 +178,11 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
         '404':
-          description: Not found. Typically in case of requesting from a non existing topic of partition.
+          description: Not found. Typically in case of requesting from a non existing topic or partition. Details are provided on the returned `Problem`.
           schema:
             $ref: '#/definitions/Problem'
         '500':
```

Comment:
503 Service Unavailable is somewhat more specific than 500 and implies that the condition is temporary. From the RFC:

> The 503 (Service Unavailable) status code indicates that the server
> is currently unable to handle the request due to a temporary overload
> or scheduled maintenance, which will likely be alleviated after some
> delay. 

At this moment one can't guarantee that any error that will happen will fit into this restriction, so 500 is preferable.